projects/www/src/styles.scss

```diff
@@ -26,6 +26,17 @@ html {
   text-size-adjust: none;
 }
 
+/* Hide scrollbars */
+::-webkit-scrollbar { 
+  display: none;
+}
+
+/* Hide scrollbars on Firefox and IE/Edge */
+html {
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```

Comment:
Instead of hiding all the scrollbars, we should get rid of the horizontal scrollbar only. Let's try to identify which element in the DOM causes the horizontal scrollbar to appear and fix its width.